### ISSUE TITLE
Add button modifier / navbar-item modifier that renders button as text, but not underlined

### ISSUE THREAD

**Issue by svdHero** [jgthms/bulma#3285]
This is about **Bulma**.

### Overview of the problem

This is about the Bulma **CSS framework**
I'm using Bulma **version** [0.9.2]
I am sure this issue is **not a duplicate**. However, it is somewhat related to #1583.

### Description

I want to add a button with an `onClick`-Action as a `navbar-item` to a `navbar-dropdown`. Hence, I want the button to be rendered like my other `navbar-item`s which are `<a class="navbar-item">Foo</a>`.

According to the documentation [here](https://bulma.io/documentation/elements/button/#colors), there are three modifiers for rendering a button without a border:

- `is-white`
- `is-text`
- `is-ghost`

However, `is-text` underlines the text, `is-ghost` colours the text and `is-white` only works for white background.

### Feature request

Add a button modifier (or navbar-item modifier???) that renders the button like the other hyperlinks in the `navbar-dropdown` and provides the appropriate hover mouse-cursor (hand with index finger pointing).

**Comment by ro31337** [comment]
No, don't close the issue. It's important as more frameworks are moving away from links to buttons for a11y reasons (for example, Rails 7).

**Comment by Umukka** [comment]
I think the only solution is making wrapping button in adiv with navbar-item class then setting background color of button same with navbar background.
`<nav class="navbar is-primary" role="navigation" aria-label="main-navigation">
	<div class="navbar-brand"></div>
	
	<div class="navbar-menu">
		<div class="navbar-start">
			<div class="navbar-item">
				<button class="button is-primary">How to Play</button>
			</div>
		</div>

		<div class="navbar-end"></div>
	</div>
</nav>`
